dataduct/data_access/connection.py

```diff
@@ -82,3 +83,35 @@ def rds_connection(database_name=None, sql_creds=None,
         cursorclass=cursorclass,
         **kwargs)
     return connection
+
+
+
+def get_postgres_config():
+    """Get postgres config from config file and return the dictionary
+    """
+    if not hasattr(config,'postgres'):
+	raise ETLConfigError('Postgres config not found')
+    return config.postgres
+
+
+
+
+@retry(CONNECTION_RETRIES, 60)
+@hook('connect_to_postgres')
+def postgres_connection(postgres_creds=None, autocommit=True,
+                        connect_timeout=30, **kwargs):
+    """Fetch a psql connection object to postgres
+    """
+    if postgres_creds is None:
+        postgres_creds = get_postgres_config()
+
+    connection = psycopg2.connect(
+        host=postgres_creds['HOST'],
```

Comment:
these don't seem to match the entries in the `example_config`. I think we need to update the config so that these get fetched correctly.